icu4j/maven/icu4j-localespi/pom.xml

```diff
@@ -20,8 +20,9 @@
 
   <licenses>
     <license>
-      <name>Unicode/ICU License</name>
-      <url>https://raw.githubusercontent.com/unicode-org/icu/master/icu4c/LICENSE</url>
+      <name>Unicode-DFS-2016</name>
+      <url>https://www.unicode.org/copyright.html</url>
```

Comment:
Hmm. This doesn't include the 3rd party licenses (thai dictionary etc). So perhaps this is not a good change.

For CLDR, I'm using a plugin to collect 3rd party licenses: https://github.com/unicode-org/cldr/pull/819/files#diff-47e6781b86a294ebbe230cf4c1399c49c7cbd4a8ca8a8cdf12a66e62fdcf4784R86-R91

Reply:
^ probably the 3rd party licenses would need to come into Maven format.